backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -81,6 +81,10 @@ public PimsAcquisitionFile GetById(long id)
                 .Include(r => r.PimsPropertyAcquisitionFiles)
                     .ThenInclude(rp => rp.Property)
                     .ThenInclude(p => p.DistrictCodeNavigation)
+                .Include(r => r.PimsAcquisitionFilePeople)
+                    .ThenInclude(rp => rp.Person)
+                .Include(r => r.PimsAcquisitionFilePeople)
+                    .ThenInclude(rp => rp.AcqFlPersonProfileTypeCodeNavigation)
```

Comment:
Return the acquisition team (Persons) as sub-entities of the GetAcquisitionFileById. 

Our developer notes for this story mentioned this is okay since create Acquisition already packages the team into the ACQ FILE